<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSignatureDlg</class>
 <widget class="QDialog" name="AddSignatureDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>462</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>715</width>
    <height>462</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>715</width>
    <height>462</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加签名</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc.qrc">
    <normaloff>:/new/prefix1/image/newMsg.png</normaloff>:/new/prefix1/image/newMsg.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>已有签名：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ui_hasName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>--------选择签名--------</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>新签名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_newName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>651</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>内容</string>
   </property>
   <widget class="QTextEdit" name="ui_context">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>651</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ui_addButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ui_closeButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../qrc.qrc"/>
 </resources>
 <connections/>
</ui>
